docs/en/sql-reference/functions/time-series-functions.md

```diff
@@ -170,3 +170,61 @@ Result:
     ]]                                                                                                                   │
 └────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
 ```
+
+## seriesAnomaliesDetectDecompose
+
+Detect anomalies in series data by decomposing it into its constituent components through [series decomposition](#seriesDecomposeSTL) and then analyzing the residual component to detect unusual patterns.
+
+**Syntax**
+
+``` sql
+seriesAnomaliesDetectDecompose(series);
+seriesAnomaliesDetectDecompose(series, threshold, seasonality, AD_method);
+```
+
+**Arguments**
+
+- `series` - An array of numeric values
+- `threshold` - A positive number to detect mild or stronger anomalies. The default is 1.5, [K value](https://en.wikipedia.org/wiki/Outlier#Tukey's_fences).
+- `Seasonality` - Represents the period for the seasonal analysis of series data. Supported values are:
+                - `-1`: Autodetect period using [seriesPeriodDetectFFT](#seriesPeriodDetectFFT). The default value.
+                - `0`: Used to skip extracting seasonal component.
+                - A positive integer representing the period of series.
+- `AD_method` - The method to use for anomaly detection on residual component. Supported values are:
+                - `tukey` : [Tukey fence](#seriesoutliersdetecttukey) test with standard 25th-75th percentile range.
+                - `ctukey`: Tukey fence test with custom 10th-90th percentile range. The default value.
+
+The number of data points in `series` should be at least twice the value of `period`.
+
+**Returned value**
+
+- An array of three arrays where the first array includes a ternary series containing (+1,-1,0) marking up/down/no anomaly respectively, the second array - anomaly score,
```

Comment:
```suggestion
- An array of three arrays where the first array includes a ternary series containing (+1,-1,0) marking up/down/no anomaly respectively,
the second array - anomaly score,
```
it will look better if it will be formatted in one way (description of every array on the new line or all of them on one line)